devel/validate-platforms

```diff
@@ -40,20 +40,41 @@ main() {
         echo "[$platform] Generating report file: $report"
         mkdir -p "$(dirname "$report")"
 
+        # Versions to track from `pip list`
+        tracked_packages=(bcbio-gff
+            biopython
+            cycler
+            evofr
+            nextstrain-augur
+            nextstrain-cli
+            numpy
+            numpyro
+            pandas
+            phylo-treetime
+            pyfastx
+            pysam
+            rethinkdb
+            scikit-learn
+            scipy
+            snakemake
+            xopen
+        )
+        tracked_packages_string=${tracked_packages[*]}
+        tracked_packages_delim=${tracked_packages_string// /|}
```

Comment:
Instead of this manipulation plus the `grep -E "'"$tracked_packages_delim"'"` construct, you can hold grep differently:

```
… | grep -Ff <(printf '%s\n' "${tracked_packages[@]}")
```